### ISSUE TITLE
Uncaught ReferenceError: shell is not defined

### ISSUE THREAD

**Issue by alexfedoseev** [cakecatz/flex-toolbar#54]
[Enter steps to reproduce below:]

1. Add `url` item.
2. Click it.

**Atom Version**: 1.3.2
**System**: Mac OS X 10.11.2
**Thrown From**: [flex-tool-bar](https://github.com/cakecatz/flex-toolbar) package, v0.7.2

### Stack Trace

Uncaught ReferenceError: shell is not defined

```
At /Users/Alex/.atom/packages/flex-tool-bar/types/url.coffee:18

ReferenceError: shell is not defined
    at Object.toolBar.addButton.callback (/Users/Alex/.atom/packages/flex-tool-bar/types/url.coffee:18:9)
    at space-pen-button.<anonymous> (/Users/Alex/.atom/packages/tool-bar/lib/tool-bar-button-view.coffee:30:19)
    at space-pen-button.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
    at space-pen-button.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "nucleus-dark-ui",
      "alex-base16-tomorrow-dark-theme"
    ],
    "packagesWithKeymapsDisabled": [
      "tool-bar"
    ]
  },
  "flex-tool-bar": {
    "showConfigButton": false
  }
}
```

### Installed Packages

```coffee
# User
Parinfer, v0.10.1
Stylus, v1.2.1
alex-base16-tomorrow-dark-theme, v0.27.0
color-picker, v2.0.14
custom-title, v1.0.1
flex-tool-bar, v0.7.2
highlight-selected, v0.11.1
language-applescript, v0.2.0
language-babel, v2.6.2
language-jade, v0.6.2
language-nginx, v0.6.1
language-slim, v0.5.0
linter, v1.11.3
linter-clojure, v1.1.1
linter-eslint, v5.2.6
nucleus-dark-ui, v0.7.2
pretty-json, v0.4.1
rest-client, v0.5.0
tool-bar, v0.1.9
vim-mode, v0.64.0
zentabs, v0.8.6

# Dev
No dev packages
```

**Comment by Xety** [comment]
Same here.

**Atom Version**: 1.4.0-beta3
**System**: Ubuntu 15.10
**Thrown From**: [flex-tool-bar](https://github.com/cakecatz/flex-toolbar) package, v0.7.2

**Comment by Xety** [comment]
Fixed with the version `v0.7.3`, thanks. :+1:

**Comment by cakecatz** [comment]
:grin:
